hybris/tests/test_hwcomposer.cpp

```diff
@@ -119,7 +119,7 @@ void HWComposer::present(HWComposerNativeWindowBuffer *buffer)
 	assert(err == 0);
 
 	err = hwcdevice->set(hwcdevice, HWC_NUM_DISPLAY_TYPES, mlist);
-	assert(err == 0);
+	//assert(err == 0);
```

Comment:
That is fine. However you should drop the line completely (maybe replacing with a comment referring to surfaceflinger then).